lldb/tools/lldb-dap/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ add_lldb_tool(lldb-dap
   Protocol/ProtocolBase.cpp
   Protocol/ProtocolTypes.cpp
   Protocol/ProtocolRequests.cpp
+  Handler/WriteMemoryRequestHandler.cpp
```

Comment:
This should go between the other `Handler/` files (which are sorted alphabetically)